django_rq/templates/django_rq/stats.html

```diff
@@ -28,6 +28,9 @@
                     <th>Active Jobs</th>
                     <th>Deferred Jobs</th>
                     <th>Finished Jobs</th>
+                    {% if display_scheduled_jobs %}
```

Comment:
Since the scheduled aren't defined on a per queue basis, we should create a separate section to display the schedulers, one scheduler per connection, similar to `FailedQueue`.

Reply:
Aren't Scheduled defined per queue? It defaults, but it can be specified. Am I misunderstanding something?

Reply:
At the moment, `Scheduler` works like `FailedQueue` in that all jobs are stored in a single Redis key for each DB.

So doing this will store jobs in the same Redis key:
```python
scheduler = Scheduler('foo', connection=connection)
scheduler.schedule(job, ...)
scheduler = Scheduler('bar', connection=connection)
scheduler.schedule(job, ...)
```
Since jobs stored in `Scheduler` or `FailedQueue` don't necessarily belong in the same queue, DjangoRQ doesn't show `FailedQueue` as a column for each queue. `Scheduler` should be treated similarly.